### ISSUE TITLE
elk 0.5 does not show in Protege Plugins

### ISSUE THREAD

**Issue by cebriggs7135** [obophenotype/cell-ontology-training#2]
@nicolevasilevsky 
I followed the instructions in the video here: https://github.com/obophenotype/cell-ontology-training/blob/master/README.md
to install Protégé 5.5 and elk 0.5 but the plugins dropdown still shows only an earlier version of elk.
I copied both the elk-protégé-0.5.0-SNAPSJOT and the pull-0.1.0.jar into the java plugins folder for Protégé:
![Screen Shot 2020-09-22 at 10 40 04 AM](https://user-images.githubusercontent.com/8284878/93899386-42099280-fcc2-11ea-9f5b-a634e9df93da.png)
![Screen Shot 2020-09-22 at 10 40 26 AM](https://user-images.githubusercontent.com/8284878/93899395-446bec80-fcc2-11ea-8264-4e7dbf35bdec.png)

**Comment by nicolevasilevsky** [comment]
Hmmm... that looks correct to me. The only thing I can think of to troubleshoot is to restart Protege, or try deleting those two files and try again.

Do you have more than one version of Protege on your computer? Also make sure you are adding the plugins and opening version 5.5.0 (and if you are not using v5.5.0, try downloading it [here](https://protege.stanford.edu/products.php#desktop-protege)

**Comment by nicolevasilevsky** [comment]
@cebriggs7135 let me know if it still does not work and we can troubleshoot further.

**Comment by cebriggs7135** [comment]
I have restarted it. Perhaps when I made an alias that confused things? Is it safe to simply delete Protege and redownload it? I'm getting it from your GitHub page for the training session this week.

**Comment by matentzn** [comment]
I think ELK 0.4.3 and ELK 0.5 cannot co-exist. You need to delete Elk 0.4.3!

If you have installed it using the Protege interface (rather than manually) you need to remove it first from a strange location: you could try looking for it after typing `open ~/.Protege` in your terminal? or search of `elk jar` in finder and select "Matches name" option?

**Comment by cebriggs7135** [comment]
@marentzn I've tried to find Elk 0.4.3 but have been unsuccessful. Where would it me?

**Comment by matentzn** [comment]
so in your Finder when you do this:

![image](https://user-images.githubusercontent.com/7070631/93924498-b9fcaa80-fd0c-11ea-91f6-e74a433b705f.png)

you dont see anything?

**Comment by nicolevasilevsky** [comment]
I have ELK 0.4.3 right now - it is called:
org.semanticweb.elk.jar

**Comment by nicolevasilevsky** [comment]
Yes, it's safe to delete Protege and re-download it, I've done it before. You'll have to recreate your preferences though, I think. We'll do this as step in the tutorial, but here is a preview of how your preferences should look:

![image](https://user-images.githubusercontent.com/6722114/93922295-90209500-fcc6-11ea-95da-449993d8aeba.png)

Note - your ID range for the CL is: 
Start: 4016000 
End: 4016499

![image](https://user-images.githubusercontent.com/6722114/93922325-99116680-fcc6-11ea-9129-4defed49ee0b.png)

**Comment by cebriggs7135** [comment]
The searches  foe elk, elk.jar, elk jar ..turn up nothing.
I'll delete Protege and try to set the preferences and maybe then put the elk 0.5.0 files in the java plugin folder

**Comment by matentzn** [comment]
if this does not work, you can also use ELk 0.4.3 in the tutorial.. its just a bit more annoying, because it is so chatty :)

**Comment by cebriggs7135** [comment]
That might have to do. But I'd really like to solve this mystery.

**Comment by nicolevasilevsky** [comment]
@cebriggs7135 I agree! I am free after 2pm PT/5pm ET today, if you'd like to hop in zoom together and we can try to troubleshoot. Or tomorrow works too

**Comment by cebriggs7135** [comment]
5:10 or after is good. 
FYI, when I set preferences as shown above, Creator value is stuck on Use username and won't let me use ORCID.
![Screen Shot 2020-09-22 at 3 28 30 PM](https://user-images.githubusercontent.com/8284878/93928646-a4758980-fce8-11ea-8435-073b7c065676.png)

**Comment by nicolevasilevsky** [comment]
Oh - good catch, I know why this is :)

You need to go to the User details tab and add your ORCID, see screenshot:
![image](https://user-images.githubusercontent.com/6722114/93928762-b1d24a00-fccf-11ea-91c9-1c14cc260e6e.png)

If you don't have an ORCID, you can register here: https://orcid.org/

**Comment by nicolevasilevsky** [comment]
Let's meet here at 5:10pm today: https://zoom.us/j/92390489056

**Comment by cebriggs7135** [comment]
See you at 5:10pm. 

I do have an ORCID profile.

**Comment by cebriggs7135** [comment]
@niIolevasilevsky I don't have your email. The invite does not include the zoom password. Will you send that soon?

**Comment by lubianat** [comment]
@cebriggs7135 I had a similar problem here. 
I  went to the folder of Protege and then to _Plugins_. In _Plugins_, I added the elk-0-5.jar file.
I  had to delete the "something-elk-0.4-something.jar" file and restart Protege, and now it is all right :) 

However, my search for "elk" worked just fine, so it might be a different issue :/
